README.md

```diff
@@ -1 +1,51 @@
-# expenses_app
\ No newline at end of file
+# expenses_app
+
+Expenses app allows users to keep track of their expenses.
+
+## Built With
+
+- Ruby on Rails
+- CSS
+- Bootstrap
+- Tailwind
+
+## Live Demo
+[Live Video](https://www.loom.com/share/48688ea19cd34cb5847dba03cd0f38a4)
+
```

Comment:
- [ ] For this project, You should be sure that the project is deployed properly, and the home and detail page works as expected on deployed version. The error mentioned by the previous code reviewer regarding the login/sign-up is still present in the deployed version of the project. You should also include a link to the deployed version of the project in your readme file. 

Kindly refer to [the project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md): :arrow_down: 
> The project should be deployed and accessible online.